### DISCUSSION TITLE
연속된 이미지 + 사운드를 통해 영상처럼 보이는 이미지 심사 가이드

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#183] [5. Q&A]
안녕하세요.
영상의 경우 심사 가이드라인에 의해 반려 사항으로 이해하여,
이미지 300장을 초당 30프레임으로 교체하여 10초의 영상처럼 만들고 사운드를 동기화하여 영상처럼 처리하는데,
이 역시 반려 사항이 될까요?
가이드라인에 따르면 이미지와 음성으로 구성된 콘텐츠이기에 문제가 없을 것으로 보입니다.
가이드 부탁드립니다.
감사합니다.

**Comment by beanyh** [comment]
네~ 이미지 슬라이드로 만든 화면은 심사를 진행후 가이드라인 기준에 문제 없다면 승인될 것 같습니다.

**Reply by strangestraw** [reply]
감사합니다~
